CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 0.10.3dev
 
 * [Fix] Remove force deleted snippets from dependent snippet's `with` (#717)
+* [Fix] Comments added in SQL query to be appended correctly when forming CTE (#886)
```

Comment:
is this accurate? I think we're now deleting comments before storing snippets

Reply:
[Fixed](https://github.com/ploomber/jupysql/pull/894/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR6)